schema/v1.1.0/api_models.yaml

```diff
@@ -300,31 +380,216 @@ classes:
           - cdp-common:cell_component_id
     annotations:
       plural: Datasets
+  DepositionAuthor:
+    mixins:
+      - IDMixin
+      - AuthorEntityMixin
+    description: Author of a deposition
+    attributes:
+      deposition:
+        range: Deposition
+        inverse: Deposition.authors
+    annotations:
+      plural: DepositionAuthors
+
   Deposition:
     mixins:
       - IDMixin
+      - CrossReferencesMixin
+      - DateStampedEntityMixin
     attributes:
-      datasets:
-        range: Dataset
-        inverse: Dataset.deposition
+      authors:
+        range: DepositionAuthor
+        inverse: DepositionAuthor.deposition
+        multivalued: true
+      alignments:
+        range: Alignment
+        inverse: Alignment.deposition
         multivalued: true
       annotations:
         range: Annotation
         inverse: Annotation.deposition
         multivalued: true
+      datasets:
+        range: Dataset
+        inverse: Dataset.deposition
+        multivalued: true
+      frames:
+        range: Frame
+        inverse: Frame.deposition
+        multivalued: true
+      tiltseries:
+        range: Tiltseries
+        inverse: Tiltseries.deposition
+        multivalued: true
       tomograms:
         range: Tomograms
         inverse: Tomogram.deposition
         multivalued: true
+      deposition_title:
+        exact_mappings:
+          - cdp-common:deposition_title
+      deposition_description:
+        exact_mappings:
+          - cdp-common:deposition_description
+      deposition_types:
+        range: DepositionType
+        inverse: DepositionType.deposition
+        multivalued: true
+    annotations:
+      plural: Depositions
+  DepositionType:
+    mixins:
+      - IDMixin
+    attributes:
+      depositions:
+        range: Deposition
+        inverse: Deposition.type
+      type:
+        range: cdp-common:deposition_types_enum
+    annotations:
+      plural: DepositionTypes
+  Frame:
+    mixins:
+      - IDMixin
+      - S3PrefixedEntityMixin
+    attributes:
+      deposition:
+        range: Deposition
+        inverse: Deposition.frames
+      per_section_parameters:
+        range: PerSectionParameters
+        inverse: PerSectionParameters.frames
+        multivalued: true
```

Comment:
Should the relationship between the frame and per-section parameters be 1:1?

Reply:
each per-section parameters would map to a frame entry thru tiltseries and z-section

Reply:
Frames can (at least technically) have multiple per-section-parameters, one for each tiltseries.